consultation_analyser/consultations/dummy_data.py

```diff
@@ -20,100 +24,162 @@
 
 logger = settings.LOGGER
 
+DATA_BY_STAGE = {
+    Consultation.Stage.ANALYSIS: {
+        "CONSULTATION_NAME": "Dummy Consultation at Analysis Stage",
+        "QUESTION_THEME_STATUS": Question.ThemeStatus.CONFIRMED,
+    },
+    Consultation.Stage.THEME_SIGN_OFF: {
+        "CONSULTATION_NAME": "Dummy Consultation at Theme Sign Off Stage",
+        "QUESTION_THEME_STATUS": Question.ThemeStatus.DRAFT,
+    },
+}
+
+
+def create_consultation(stage):
+    """Create and return a Consultation."""
+    name = DATA_BY_STAGE[stage]["CONSULTATION_NAME"]
+    return ConsultationFactory(title=name, stage=stage)
+
+
+def create_respondents(consultation, number_respondents):
+    """Create and return a list of Respondents."""
+    return [
+        RespondentFactory(consultation=consultation, themefinder_id=i + 1)
+        for i in range(number_respondents)
```

Comment:
```suggestion
RespondentFactory(consultation=consultation, themefinder_id=i)        for i in range(1, number_respondents)
```
very nitpicky! forgive me

Reply:
Just to confirm, the suggested change is from `themefinder_id=i + 1` to `themefinder_id=i`? I can do that!